build.gradle

```diff
@@ -105,8 +105,10 @@ subprojects {
 
         libraries = [
                 auto_value     : "com.google.auto.value:auto-value:${autoValueVersion}",
+                byte_buddy   : 'net.bytebuddy:byte-buddy:1.7.1',
                 disruptor    : 'com.lmax:disruptor:3.3.6',
                 errorprone   : 'com.google.errorprone:error_prone_annotations:2.0.11',
+                findbugs_annotations : "com.google.code.findbugs:annotations:${findBugsVersion}",
```

Comment:
Good question. In the grpc-java repo, I see that findbugs jsr305 is used (version 3.0.0), https://github.com/grpc/grpc-java/search?q=findbugs, but no other reference to findbugs (the tool or the annotations for suppressing FB warnings), this puzzles me a bit, but well....

WRT JSR305, there don't seem to be any API-relevant changes between 3.0.0 and 3.0.1 (https://abi-laboratory.pro/java/tracker/timeline/jsr305/). Since opencensus-java is using tool version 3.0.1 (the current version), I figured that it would be best to use the same version for jsr305 and annotations. I'm leaving this as is for now, but happy to send a follow-up PR. PLKM.